### ISSUE TITLE
False positive: "format_compact_decimal" is unknown import symbol (reportGeneralTypeIssues)

### ISSUE THREAD

**Issue by DenverCoder1** [microsoft/pyright#4138]
**Describe the bug**

Pyright reports an unknown import symbol, even though the symbol exists and the version is correct.

**To Reproduce**

Ensure the Python package `Babel==2.11.0` is installed ("format_compact_decimal" is only in `Babel>=2.11.0`)

The following line causes a Pyright error:
```py
from babel.numbers import format_compact_decimal
```
```
Error: "format_compact_decimal" is unknown import symbol (reportGeneralTypeIssues)
```

**Expected behavior**

This should not report the error since the symbol exists and the code runs properly.

**Screenshots or Code**

![image](https://user-images.githubusercontent.com/20955511/200056258-263cd4b9-2807-4b00-b37f-30e9d0365d11.png)

`Babel==2.11.0` is required and I have made extra sure this is the version that is installed.

More context from github action output - https://github.com/DenverCoder1/github-readme-youtube-cards/actions/runs/3396394994/jobs/5647413982

Same occurs in VS Code with Pylance.

The latest version of Pyright is used (pyright 1.1.278)

**Comment by erictraut** [comment]
Babel is an untyped library. Its types come from typeshed stubs, which are bundled with pyright. The symbol `format_compact_decimal` is not included in the typeshed stubs for babel. If you think it should be included, please file a bug in the [typeshed project](https://github.com/python/typeshed) — or better yet, submit a PR with the associated change. Pyright regularly (every couple of weeks) pulls the updated stubs from typeshed, so any improvements made there will be reflected in pyright.

**Comment by DenverCoder1** [comment]
Thanks for the explanation! The feature was only added in the past few days, so it makes sense that's where the issue is.
